notebooks/03-query/16-export-msq-query.ipynb

```diff
@@ -0,0 +1,426 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a474e38b-07aa-4f7e-9275-2a2cd18ac0fb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!rm -rf /opt/shared/exports/example-wiki-export\n",
+    "!mkdir -p /opt/shared/exports/example-wiki-export"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "96a9b601-dfa6-48a3-a00b-e710a812d2e2",
+   "metadata": {},
+   "source": [
+    "The next cell executes the the export:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0b36cb33-1e8a-4284-966c-a3937c8a33a4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sql='''\n",
+    "    INSERT INTO \n",
+    "       EXTERN ( local(exportPath => '/opt/shared/exports/example-wiki-export') )\n",
+    "       AS CSV\n",
+    "    SELECT\n",
+    "      *\n",
+    "    FROM \"example-wiki-export\" \n",
+    "'''\n",
+    "display_client.run_task(sql)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f9571de0-eac8-40f0-8970-f137aec9e767",
+   "metadata": {},
+   "source": [
+    "Take a look at the file this command created, notice that the name contains the query id, the worker task that created the file and a partition number, each MSQ worker creates one or more partitions:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a1cef199-430d-4094-b006-090d1013af7b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!ls /opt/shared/exports/example-wiki-export"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1580ec6e-3172-4829-97bc-53b71c72ec0e",
+   "metadata": {},
+   "source": [
+    "You can see the content of the file with:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "95265824-7962-4dd6-a93f-1a6281633c53",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!head /opt/shared/exports/example-wiki-export/*"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8dfe9b15-0724-419a-ad97-1a35971f1f47",
+   "metadata": {},
+   "source": [
+    "## Exporting transformed data\n",
+    "Export works for any SQL SELECT statement.\n",
+    "\n",
+    "Run following cell to export aggregate query results:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2c044cdb-4631-4e3f-9552-4cedc0b598d7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!rm -rf /opt/shared/exports/example-wiki-export-agg\n",
+    "!mkdir -p /opt/shared/exports/example-wiki-export-agg\n",
+    "\n",
+    "sql='''\n",
+    "INSERT INTO \n",
+    "       EXTERN ( local(exportPath => '/opt/shared/exports/example-wiki-export-agg') )\n",
+    "       AS CSV\n",
+    "\n",
+    "SELECT \"user\" as \"user\",\n",
+    "       \"channel\" as \"channel\",\n",
+    "       SUM(\"added\"+\"deleted\") as \"total_changes\"\n",
+    "FROM \"example-wiki-export\"\n",
+    "GROUP BY 1,2\n",
+    "'''\n",
+    "display_client.run_task(sql)"
+   ]
+  },
```

Comment:
Here, the [destination function](link to docs somewhere??) will cause Druid to output the file to the "example-wiki-export-agg" folder.

Run the following command to see the first XXXXX rows from the export.

Reply:
done